app/services/puppetdb_host.rb

```diff
@@ -26,7 +26,7 @@ def parse_facts(pdb_facts)
   end
 
   def parse_fact_value(value)
-    result = JSON.parse(value)
+    result = value.is_a?(String) ? JSON.parse(value) : value
```

Comment:
[Puppet supports non-string facts](https://docs.puppet.com/puppet/4.10/lang_facts_and_builtin_vars.html#data-types) (boolean for example), so in the case of `JSON.parse(false)` we would get a `TypeError`,  in order to avoid this, it's safe to JSON parse only strings.